apps/client/src/domains/map/Map.tsx

```
@@ -0,0 +1,9 @@
+import { useMap } from '@/shared/services';
+
+import s from './Map.module.css';
+
+export const Map = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)